manifests/bosh-manifest/deployments/010-bosh-template.yml

```diff
@@ -81,7 +85,7 @@ jobs:
     ntp: (( grab meta.ntp ))
 
     agent:
-      mbus: (( concat "nats://nats:" secrets.bosh_nats_password "@" meta.bosh_service_ip ":4222" ))
```

Comment:
We should also remove `meta.bosh_service_ip` from the manifest if it's no longer used.